pages/common/dd.md

```diff
@@ -1,28 +1,24 @@
 # dd
 
 > Convert and copy a file.
-> More information: <https://www.gnu.org/software/coreutils/dd>.
+> More information: <https://manned.org/man/dd.1p>.
 
-- Make a bootable USB drive from an isohybrid file (such like `archlinux-xxx.iso`):
+- Make a bootable USB drive from an isohybrid file (such as `archlinux-xxx.iso`):
 
 `dd if={{path/to/file.iso}} of=/dev/{{usb_drive}}`
 
-- Clone a drive to another drive with 4 MiB block and ignore error:
+- Clone a drive to another drive with 4 MiB block size and flush writes before the command terminates:
 
-`dd if=/dev/{{source_drive}} of=/dev/{{dest_drive}} bs={{4194304}} conv={{noerror}}`
+`dd bs={{4194304}} conv={{fsync}} if=/dev/{{source_drive}} of=/dev/{{dest_drive}}`
 
 - Generate a file of 100 random bytes by using kernel random driver:
```

Comment:
```suggestion
 - Generate a file with a specific number of random bytes by using kernel random driver:
```
As "100" is a placeholder, we need to leave the description generic.

Feel free to reword it.